redirects.js

```diff
@@ -585,6 +585,10 @@ const userDocsRedirects = [
       '/platforms/:platform/guides/:guide/sourcemaps/:section(generating|validating|best-practices|artifact-and-release-bundles)/',
     destination: '/platforms/:platform/guides/:guide/sourcemaps/',
   },
+  {
+    source: '/platforms/javascript/sourcemaps/troubleshooting_js/artifact-bundles/',
+    destination: '/platforms/javascript/sourcemaps/troubleshooting_js/debug-ids/',
+  },
```

Comment:
This was the only change I made to the file, the rest was likely due to my code formatter